thesis.cls

```diff
@@ -192,7 +192,9 @@
 % Chapter heading
 \def\@chapter[#1]#2
 {
-
+	% the footnotes should start with 1 in each chapter
+	\setcounter{footnoteA}{0}
```

Comment:
I don't understand why this is set to 0 if the comment says it should start at 1.